docs/install/build_from_source.md

```diff
@@ -317,6 +317,40 @@ Use [CMake](https://cmake.org/) to create a Visual Studio solution in ```./build
 In Visual Studio, open the solution file,```.sln```, and compile it.
 These commands produce a library called ```mxnet.dll``` in the ```./build/Release/``` or ```./build/Debug``` folder.
 
+</div>
+
+<div class="linux ubuntu">
+
+## Build MXNet using NCCL
+- Download and install the latest NCCL library from NVIDIA.
+- Note the directory path in which NCCL libraries and header files are installed.
+- Ensure that the installation directory contains ```lib``` and ```include``` folders.
+- Ensure that the prerequisites for using NCCL such as Cuda libraries are met. 
+- Append the ```config.mk``` file with following, in addition to the CUDA related options.
+- USE_NCCL=1
+- USE_NCCL_PATH=path-to-nccl-installation-folder
+``` bash
+echo "USE_NCCL=1" >> make/config.mk
+echo "USE_NCCP_PATH=path-to-nccl-installation-folder" >> make/config.mk
+cp make/config.mk .
+```
+- Run make command
+``` bash
+make -j"$(nproc)"
+```
+
+## Validation
+- Follow the steps to install MXNet Python binding.
+- Comment the following line in ```test_nccl.py``` file at ```incubator-mxnet/tests/python/gpu/test_nccl.py```
+``` bash
+@unittest.skip("Test requires NCCL library installed and enabled during build")
+```
+- Run test_nccl.py script as follows. The test should complete. It does not produce any output.
```

Comment:
`Run test_nccl.py script as follows` -> Should there be a command to show the users how to run the test?
Can we also print "passed" at the end of the unit test instead of not showing any output, since it will be run by users?  @ptrendx

Reply:
The line to run the test should be
```nosetests --verbose tests/python/gpu/test_nccl.py```
Adding "passed" at the end is just a single line at the end of the `test_nccl_pushpull` function. Do you want me to do it in separate PR or will you do it in this PR @leleamol?

Reply:
Please add it in this PR to avoid fragmentation. No need to make a subsequent PR if this one is already open.

Reply:
I will add the "Passed" line at the end of this script in this PR